packages/docs/components/ColorCards/availableColors.ts

```diff
@@ -0,0 +1,105 @@
+import { ThemeInterface } from '@bigcommerce/big-design-theme';
+
+type Colors = ThemeInterface['colors'];
+type Color = keyof Colors;
+
+export interface ColorDetails {
+  contrast?: 'AAA' | 'AA' | 'A';
+  description?: {
+    text: string;
+    href?: string;
+  };
+  name: Color;
+  textColor: Color;
+}
+
+export const accentColors: ColorDetails[] = [
+  { contrast: 'AA', name: 'primary40', textColor: 'white' },
+  { contrast: 'AAA', name: 'primary10', textColor: 'secondary70' },
+  { contrast: 'AAA', name: 'primary20', textColor: 'secondary70' },
+  { contrast: 'AAA', name: 'primary30', textColor: 'secondary70' },
+  { contrast: 'AA', name: 'primary50', textColor: 'white' },
+  { contrast: 'AAA', name: 'primary60', textColor: 'white' },
+  { contrast: 'AAA', name: 'primary70', textColor: 'white' },
+];
+
+export const typeColors: ColorDetails[] = [
+  { contrast: 'AAA', name: 'secondary70', textColor: 'white' },
+  { contrast: 'AA', name: 'secondary60', textColor: 'white' },
+  { contrast: 'AA', name: 'secondary50', textColor: 'white' },
+  { contrast: 'AAA', name: 'white', textColor: 'secondary70' },
+];
+
+export const fillsAndStrokesColors: ColorDetails[] = [
+  { contrast: 'AAA', name: 'white', textColor: 'secondary70' },
+  { contrast: 'AAA', name: 'secondary10', textColor: 'secondary70' },
+  { contrast: 'AA', name: 'secondary20', textColor: 'secondary70' },
+  { contrast: 'A', name: 'secondary30', textColor: 'secondary70' },
+  { contrast: 'A', name: 'secondary40', textColor: 'secondary70' },
```

Comment:
I omitted color `secondary70(40%)` since I think we don't have this type of color in `Big-design`. 🤔  We just have `secondary70` without the 40%.